src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -42,6 +42,8 @@
     protected boolean sab; // If shooting SAB right now
     protected boolean rsb; // If shooting RSB right nos
 
+    protected boolean closeRange; // Ignore radius before first shot
```

Comment:
```suggestion
    protected boolean closeRange; // If we've been close enough to shoot this npc yet
````
Update this comment, the variable is not about ignoring, the variable is only about keeping track of if we've been close enough